src/algebra/add_torsor.lean

```diff
@@ -80,29 +84,27 @@ instance add_group_is_add_torsor (G : Type*) [add_group G] :
 
 /-- Simplify addition for a torsor for an `add_group G` over
 itself. -/
-@[simp] lemma vadd_eq_add (G : Type*) [add_group G] (g1 g2 : G) : g1 +ᵥ g2 = g1 + g2 :=
+@[simp] lemma vadd_eq_add {G : Type*} [add_group G] (g1 g2 : G) : g1 +ᵥ g2 = g1 + g2 :=
 rfl
 
 /-- Simplify subtraction for a torsor for an `add_group G` over
 itself. -/
-@[simp] lemma vsub_eq_sub (G : Type*) [add_group G] (g1 g2 : G) : g1 -ᵥ g2 = g1 - g2 :=
+@[simp] lemma vsub_eq_sub {G : Type*} [add_group G] (g1 g2 : G) : g1 -ᵥ g2 = g1 - g2 :=
 rfl
 
-namespace add_action
-
 section general
 
 variables (G : Type*) {P : Type*} [add_monoid G] [A : add_action G P]
 include A
 
 /-- Adding the zero group element to a point gives the same point. -/
 @[simp] lemma zero_vadd (p : P) : (0 : G) +ᵥ p = p :=
-zero_vadd' p
+add_action.zero_vadd' p
 
 /-- Adding two group elements to a point produces the same result as
 adding their sum. -/
 lemma vadd_assoc (g1 g2 : G) (p : P) : g1 +ᵥ (g2 +ᵥ p) = (g1 + g2) +ᵥ p :=
-vadd_assoc' g1 g2 p
+add_action.vadd_assoc' g1 g2 p
```

Comment:
I don't think this actually needed `G` as an explicit argument even before this change; it certainly shouldn't need it as an explicit argument afterwards.

Reply:
Feel free to push to this branch.